### PR TITLE
Feature/lu 6127 selection file lambda

### PR DESCRIPTION
**Pull request by larissanne**

### CODE REVIEW

File: aws-infrastructure/ingest.tf [hunk 1]
```diff
@@ -451,13 +463,46 @@ resource "aws_lambda_function" "bpmtransform" {
...
+}
+
 resource "aws_lambda_function" "idbrtojson" {
 
   function_name = "${local.name_prefix}-idbrtojson"
   role = aws_iam_role.lambda.arn
   handler = "handler.lambda_handler"
   runtime = "python3.7"
   filename = data.archive_file.dummy.output_path
+  # layers = [aws_lambda_layer_version.python_layer.arn]
```
Comment (RyanAldy) [inline aws-infrastructure/ingest.tf:505]: Delete this

File: lambda-ingest-selection-file/pick_up_selection.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import requests
+
+# read files in s3 bucket
+def read_file_from_s3(json_filename):
+    ingest_zone_bucket = os.getenv("INGEST_ZONE_BUCKET")
+    ingest_zone_filename = json_filename
+    s3 = boto3.resource("s3")
+    obj = s3.Object(ingest_zone_bucket, ingest_zone_filename)
+    body = obj.get()["Body"].read().decode("UTF-8")
+    #formtype = body[Values][formtype]
```
Comment (RyanAldy) [inline lambda-ingest-selection-file/pick_up_selection.py:13]: Delete commented lines

File: lambda-ingest-selection-file/pick_up_selection.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+def load_selection_file(period, survey, data_to_business_layer):
+    lambdaName = "Load-selection-file: "
+    business_layer_endpoint = os.getenv("BUSINESS_LAYER_ENDPOINT")
+    request_response = ''
+    try:
+        request_response = requests.post(business_layer_endpoint, data_to_business_layer)
+        print('Response: ' + str(request_response))               
+    except Exception as error:
+        errorMessage = lambdaName + " Problem with call to Business Layer " + str(error)
+        # error_queue(errorMessage)
```
Comment (RyanAldy) [inline lambda-ingest-selection-file/pick_up_selection.py:39]: We probably need to write the error message somewhere, so we'll need to add this function in and the code for it

File: lambda-ingest-selection-file/pick_up_selection.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        print(errorMessage)
+        print('Response: ' + str(request_response))
+
+# write notification of success/failure to queue
```
Comment (RyanAldy) [inline lambda-ingest-selection-file/pick_up_selection.py:43]: May need this too?

File: pipeline.yml [hunk 5]
```diff
@@ -165,6 +165,40 @@ resources:
...
+
+###################################################################################################
+# ####### lambda-ingestion-selection-file
+###################################################################################################
+
+  - name: takeon-ingestion-selection-file-git-repo
+    type: git
+    source:
+      paths:
+        - lambda-ingest-tselection-file/
```
Comment (RyanAldy) [inline pipeline.yml:177]: ingest-tselection, is that intentional?

**Review by RyanAldy — CHANGES_REQUESTED** [review]

**Review by RyanAldy — APPROVED** [review]

**Comment by ranjit-das** [comment]
Tested and found that contributor is inserted into db from selection file

**Pull request merged**